src/engine/enginemaster.cpp

```diff
@@ -155,14 +171,17 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
     m_pKeylockEngine->set(pConfig->getValueString(
             ConfigKey(group, "keylock_engine")).toDouble());
 
+    // TODO: Make this read only but let EngineMasterTest enable it
```

Comment:
There is no need for the user to be concerned with this. Mixxx should detect when processing the master mix is necessary and disable it automatically if it is not needed.